### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by juhamust**
- More informatic: Easier to get started with widgets
- More up to date with current status
- More engaging visual example

**Notes:**
- Image path needs to be fixed once merged
- See the rendered outcome: https://github.com/juhamust/mozaik/tree/readme
- My motivation for the change is get more contributors for the project/widgets

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,42 +5,116 @@
...
 [![Widgets][widget-count-image]][widget-count-url]
 
-Mozaïk is a tool based on nodejs / react / d3 / stylus to easily craft beautiful dashboards.
-It ships with several useful [widgets](https://github.com/plouc/mozaik/wiki/widgets) (for now, focused on CI and monitoring) such as Jenkins, Travis, Sensu…
-You can switch between 5 available themes or [make your own one](https://github.com/plouc/mozaik/wiki/theming).
-One of its handy feature is to be able to make a *rotation* between several dashboards with a single [config file](https://github.com/plouc/mozaik/wiki/configuration).
+Mozaïk is a tool based on nodejs / react / reflux / d3 / stylus to easily craft beautiful dashboards. [See demo](http://mozaik.herokuapp.com/)
 
-![preview](https://raw.githubusercontent.com/wiki/plouc/mozaik/assets/mozaik-panorama.png)
+![preview](https://raw.githubusercontent.com/juhamust/mozaik/readme/preview.png)
```
Comment (plouc) [inline README.md]: You should put the screenshot in the same repo

**Comment by plouc** [comment]
Thabks a lot for your contribution, I hope it will help people contributing!

File: README.md (hunk 1 above)
Comment (plouc) [inline README.md]: Didn't saw your commrnt on paths

Comment (juhamust) [inline README.md]: I would have included the screenshot in PR, but to be honest, I don't know how to do that with wiki.

**Pull request merged**